### ISSUE TITLE
[Feature Request] New field validation option: If exists (in DB)

### ISSUE THREAD

**Issue by brettnpath** [JhumanJ/OpnForm#619]
It would be fantastic if we could build into the frontend a couple of new validation options on a per-field basis. Effectively, if the order number already exists in a response (in the database) then don't allow the form to be filled again.

In this example, the order number is unique, and the form should only ever be filled once per order number.

Hosted docker instance.

![image](https://github.com/user-attachments/assets/138f9c4b-0ac0-4eb6-b00b-b272a8cc6a82)

**Comment by SmartPhoneLover** [comment]
This would be a great implementation, I would also need it.

**Comment by JonasTrampe** [comment]
I need this, too!

**Comment by satellouk** [comment]
Yep, just wondering how to avoid double entries based on a field such as email address for example. UP.
